### ISSUE TITLE
Version 72 support would be amazing 

### ISSUE THREAD

**Issue by Marmelador** [duraki/SketchCrapp#81]
**Sketch version [required]**
What is the Sketch version you want support for?
Ex: 70

**Additional context [optional]**
Would appreciate it. Thanks for the good work

**Comment by elijahtsai** [comment]
We are working on it!!🛠🛠🛠

**Comment by Suriv** [comment]
The app has been updated to 72.1

**Comment by chenqingze** [comment]
The app has been updated to 72.2

**Comment by Suriv** [comment]
The app has been updated to 72.3

**Comment by terence-codigo** [comment]
72.4 is out.

**Comment by squeeish** [comment]
72.4 is out.

**Comment by jpransxdev** [comment]
Is this still being worked on?

**Comment by duraki** [comment]
@jpransxdev yes, we are just a tad more busy then usual so expect delays. Not to say we are having other cracks we are keeping alive. Would you guys be interested in Telegram/Signal group to stay in sync for all MacOS cracks?

**Comment by jpransxdev** [comment]
Yes actually I would be interested. Telegram preferred.

**Comment by jpransxdev** [comment]
73 is out

**Comment by jpransxdev** [comment]
73.1 Is out

**Comment by jpransxdev** [comment]
74.1 is out

**Comment by jpransxdev** [comment]
75.0 is out!

**Comment by jpransxdev** [comment]
Sketch 75 is out

**Comment by Suriv** [comment]
Is there a way to update the repositories for the various versions? I say it to lend a hand to update it

**Comment by duraki** [comment]
@Suriv post offsets here and we'll credit you. Or make a pull request and we'll see to accept it. Regards.

**Comment by Suriv** [comment]
Ok.
Now I am missing the parameters to add for the various versions.

Can you find out where you can get it or do you have to talk on a Telegram channel?

**Comment by elijahtsai** [comment]
Hello Everyone. First of all thanks for update to us about the news of every new update and supported us. But since Sketch change the way of verifying the trial our existing method is useless start from v72 and I don't know how to crack it on their new way after I tried a lot of other method 🥺. So if you have come up with any ideas or methods, you can share at here or ask us how to update the script or best make a pull request, we are all welcome to contribute. Many thanks, @elijahtsai
